src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,7 +68,10 @@ public ReadOnlyPerson getPerson() {
     @Override
     public CommandResult execute() {
         try {
+        	System.out.println("THis executed");
             addressBook.addPerson(toAdd);
+            //CommandResult commandresult = new ListCommand().execute();
```

Comment:
Remove code not in use instead of commenting it out. You are using RCS so you can rollback to previous versions with easy.